test/JDBC/input/pg_hint_plan/BABEL-3513-vu-verify.mix

```diff
@@ -6,6 +6,9 @@ analyze master_dbo.babel_3513_t2;
 go
 
 -- tsql
+exec sp_babelfish_configure 'enable_pg_hint', 'on', 'server'
+go
```

Comment:
Why this change?

Reply:
Oh this is needed because we don't preserve the database settings across dump/restore, so we basically need to reset the value of `enable_pg_hint`.

Reply:
Got it. But then won't we be missing coverage where it should be preserved  in case of M/mVU.